Mark9Mbugua/Politico-V
# #163782515 Create repo structure

#### What does this PR do?
It creates the repository structure

#### Description of Task to be completed?
- Install a virtual environment in the root directory
- Ensure all the files and folders that are necessary for the creation and  completion of the project exist and are in good order
- Install project requirements
- set up .gitignore
- set up run.py

#### How should this be manually tested?
- Clone this repo in a local repo
- Write tests for endpoints that fail at first, create the endpoints and ensure that the tests pass this time
- Flask run the application
-Using postman, see if the endpoints work

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2243191
#163782515
